### DISCUSSION TITLE
.b13: Brouter maps not available in Cruiser download

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#400] [Q&A] [answered]
The internal download of Brouter map files didn't work on my 2nd device (Android 13), not used for a while.
It showed no segements to download. When I use Brouter app, then my default 9 maps are shown and can be downloaded.

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

What do you mean "It showed no segments to download"?

Do you use the BRouter downloader inside Cruiser?
Cruiser map downloader?
"Check available routing data" menu?

**Reply by devemux86** [reply]
@DolomitesTom please write the exact steps you are doing so we can understand what happens.

**Reply by DolomitesTom** [reply]
> Do you use the BRouter downloader inside Cruiser?
Cruiser map downloader?

What's the difference? I use menu -- downloads -- Brouter. I guess the marked rectangles for map download got lost at some point and I didn't recognize it till now. On my main device there were only 2 segments of 9 still selected for updating. It is strange, I didn't change anything here. I had the 9 segments for central Europe selected since years. In brouter app they were still marked. I selected them again in Cruiser and then they were downloaded.

> "Check available routing data" menu?

Sorry don't know what you mean with this.

**Reply by devemux86** [reply, marked as answer]
> I had the 9 segments for central Europe selected since years.

This is the difference, you used the external BRouter app.

The default BRouter on the new Cruiser is the internal BRouter (_with its own downloads_).
It can be controlled in advanced settings of BRouter.

Internal BRouter uses newer code than the current version of BRouter app on Google Play.

---

"Check available routing data" is done automatically in route calculations with internal BRouter,
or can be done manually via an option in the long press menu, please see here:

- https://github.com/devemux86/cruiser/discussions/380#discussioncomment-7672452

**Reply by DolomitesTom** [reply]
OK was not aware of this, thanks
